libsolidity/ast/ASTJsonExporter.cpp

```diff
@@ -293,7 +302,8 @@ bool ASTJsonExporter::visit(ContractDefinition const& _node)
 		std::make_pair("usedEvents", getContainerIds(_node.interfaceEvents(false))),
 		std::make_pair("usedErrors", getContainerIds(_node.interfaceErrors(false))),
 		std::make_pair("nodes", toJson(_node.subNodes())),
-		std::make_pair("scope", idOrNull(_node.scope()))
+		std::make_pair("scope", idOrNull(_node.scope())),
+		std::make_pair("storageLayoutSpecifier", _node.storageLayoutSpecifier() ? toJson(*_node.storageLayoutSpecifier()) : Json())
```

Comment:
On the other one we can probably shorten `storageLayoutSpecifier` without much loss:
```suggestion
		std::make_pair("storageLayout", toJson(_node.storageLayoutSpecifier()))
```